addons/binding/org.openhab.binding.kostal.inverter/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,67 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="kostal"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+        xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+    <thing-type id="kostal-inverter">
+        <label>Kostal Inverter</label>
+        <description>Kostal Inverter</description>
+        <channels>
+          <channel typeId="acPower" id="acPower" ></channel>
+          <channel typeId="totalEnergy" id="totalEnergy" ></channel>
+          <channel typeId="dayEnergy" id="dayEnergy" ></channel>
+          <channel typeId="status" id="status" ></channel>
+        </channels>
+
+        <config-description>
+			<parameter name="url" type="text" required="true">
+				<label>URL of the web interface</label>
+				<default>http://192.168.0.x</default>
```

Comment:
I would prefer default values to be valid - maybe you can add this as a description instead.